pyracing/response_objects/historical_data.py

```diff
@@ -269,7 +269,7 @@ def __init__(self, dict):
 
 class Team:
     def __init__(self, dict):
-        self.cars = dict['cars']
+        self.cars = parse_encode(dict['cars'])
```

Comment:
Is this also a string of comma separated numbers?

Reply:
It comes back in this ugly mess: "%5B%7B%22carclassid%22%3A473%2C%22carid%22%3A73%7D%5D"